integration/test-executions.js

```diff
@@ -12,7 +12,7 @@ var expect = chai.expect
   , st2client = require('../index')(config)
   ;
 
-var MINIMUM_ENTITIES = 3;
+var MINIMUM_ENTITIES = 1;
```

Comment:
The reason why I set this to 1 is because if I test this on a brand new installation, this fails. So if the purpose is to check >= 3 items, then will have to run several executions here somewhere.